ops/storage-ddls/cassandra-setup.cql

```diff
@@ -57,8 +57,6 @@ CREATE TABLE trustedsources (
    PRIMARY KEY (pipelinekey, externalsourceid, sourcetype, rank)
 );
 
-DROP TABLE IF EXISTS events;
-DROP TYPE IF EXISTS features;
```

Comment:
These drop statements need to be here. If you put them next to their `CREATE TABLE` counterparts, the `DROP TABLE` of their referencing table(s) will fail. For instance, the `DROP TABLE features` will fail if we don't re-arrange the order in which it's dependency tables are added.

Having said that, I agree that having three seemingly-random DROP TABLE statements looks weird. Can I suggest we move the `CREATE TABLE` AND `CREATE TYPE` statements instead?